monkey/agent_plugins/credentials_collectors/chrome/src/database_reader.py

```diff
@@ -40,6 +40,6 @@ def _extract_login_data(database_path: Path) -> Iterator[ExtractedCredentialPair
         for user, password in conn.execute(DB_SQL_STATEMENT):
             yield user, password
     except Exception:
-        logger.exception("Error encounter while connecting to " f"database: {database_path}")
+        logger.exception(f"Encountered an error while connecting to database: {database_path}")
```

Comment:
This could be an error while connecting Or querying. Either catch specific exceptions or Write a more generic error message. You could include the exception string in the generic error message for clarity.

Reply:
Do we need to see a stack trace when handling this?